src/balance.ts

```diff
@@ -0,0 +1,78 @@
+import { setupClient } from './setup_client.ts'
+import { Address} from "@polycrypt/erdstall/ledger";
+import { Client } from '@polycrypt/erdstall';
+import { widget } from './widget.ts';
+
+/**
+ * export function to change to balance viewer
+ * @param html main body of widget
+ */
+export async function htmlBalance(html : HTMLDivElement) {
+  html.innerHTML = `
+  <div>
+    <a href="https://github.com/perun-network/erdstall-ts-sdk" target="_blank">
+      <img src="https://nifty.erdstall.dev/static/media/erdstall-logo.4ca5436f.png" class="logo vanilla" alt="TypeScript logo" />
+    </a>
+    <h1>erdPay</h1>
+    <div class="card">
+      <button id="balance" type="button"></button>
+      <button id="back" type="button"></button>
+    </div>
+    <div>
+        <label for="address">Address:</label>
+        <input type="text" id="address" name="address">
+    </div>
+    <div>
+        <label id="array"></label>
+    </div>
+  </div>
+  `
+
+  // Try to set up client
+  let client
+  try {
+    client = await setupClient()
+  } catch (error) {
+    alert(error)
+  }
+
+  const back = document.querySelector<HTMLButtonElement>('#back')!
+  const button = document.querySelector<HTMLButtonElement>('#balance')!
+  let input = document.querySelector<HTMLInputElement>('#address')!
+  const array = document.querySelector<HTMLBodyElement>('#array')!
+
+  // initialize buttons
```

Comment:
More easily recognizable identifier like:

goBackBtn
viewBalanceBtn
inputAddressTxt
assetArray

Reply:
Will be fixed next iteration. (Ongoing in branch "design")